### PR TITLE
add september heartcore release notes. update redirect api with new endpoint

### PR DESCRIPTION
**Pull request by jmf-umbraco**
## Description

Heartcore release notes & updated API docs for September.

Also moved the redirect API to underneath the content delivery API - It's not really a seperate API on its own and the docs should reflect that.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Heartcore

## Deadline (if relevant)

_When should the content be published?_

N/A

### CODE REVIEW

File: umbraco-heartcore/release-notes/2024-08-releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# August 2024
+The following changes have been released to all Heartcore sites.
+
+## Webhook Enhancements
+With the core CMS having added webhook support, we have adapted the same function in Heartcore to use the same user interface. You'll now get the same consistent editing experience in both core Umbraco and Heartcore, but also some new features!
```
Comment (sofietoft) [inline umbraco-heartcore/release-notes/2024-08-releasenotes.md:5]: We try to not use third person (we). Also, I've rewritten this a bit according to Grammarly recommendations.
Let me know if something doesn't make sense!

```suggestion
With the core CMS adding webhook support, the same functionality has been adapted into Heartcore to use the same user interface. You will now get the same consistent editing experience in both Umbraco CMS and Umbraco Heartcore, and it also comes with new features.
```

File: umbraco-heartcore/release-notes/2024-08-releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+* You may now select multiple event types (for example, publish / unpublish) or content types for a webhook. No more needing to create dozens of hooks to achieve what can now be done with only one!
```
Comment (sofietoft) [inline umbraco-heartcore/release-notes/2024-08-releasenotes.md:6]:
```suggestion
* You may now select multiple event types (for example, publish / unpublish) or content types for a webhook. No more needing to create dozens of hooks to achieve what can now be done with only one.
```

File: umbraco-heartcore/release-notes/2024-08-releasenotes.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+* You may now specify custom HTTP headers to send to your endpoint whenever the webhook fires.
+
+Aside from the interface, all other aspects of webhooks in Heartcore are unchanged. Webhooks are still fired from the delivery platform with the same IP address range, retry policy, and payload that you're used to.
```
Comment (sofietoft) [inline umbraco-heartcore/release-notes/2024-08-releasenotes.md:9]:
```suggestion
Aside from the interface, all other aspects of webhooks in Heartcore are unchanged. Webhooks are still fired from the delivery platform with the same IP address range, retry policy, and payload that you are used to.
```

File: umbraco-heartcore/release-notes/2024-08-releasenotes.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+For more information about webhook functionality, both new and old, check out the [updated page here](../getting-started/webhooks.md).
```
Comment (sofietoft) [inline umbraco-heartcore/release-notes/2024-08-releasenotes.md:11]:
```suggestion
For more information about webhook functionality, both new and old, check out the [updated Webhooks article](../getting-started/webhooks.md).
```

File: umbraco-heartcore/release-notes/2024-08-releasenotes.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+## Improved Firewall
+We can't talk too much about this one, but we recently rolled out additional firewall rules for all Heartcore services. While we have made an effort to tune these, there is always a small possibility of false positives. If you believe your legitimate traffic is affected, please raise a support ticket through the portal.`
```
Comment (sofietoft) [inline umbraco-heartcore/release-notes/2024-08-releasenotes.md:14]:
```suggestion
A recent roll-out as added additional firewall rules for all Heartcore services. While efforts have been made to tune these, there is always a small possibility of false positives. If you believe your legitimate traffic is affected, please raise a support ticket through the portal.
```

File: umbraco-heartcore/release-notes/2024-09-releasenotes.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# September 2024
+The following changes have been released to all Heartcore sites.
+
+## New Redirect Endpoint
+For a long time, the only way to get redirect information from Heartcore was a single endpoint on the Delivery API. This endpoint returned paginated redirect information for _all_ content items in the tree. If you needed to find the canonical URL for a given path, then this endpoint was your only option. You would need to find your path in the returned items - especially challenging if it wasn't on the first page of results!
```
Comment (sofietoft) [inline umbraco-heartcore/release-notes/2024-09-releasenotes.md:5]:
```suggestion
For a long time, the only way to get redirect information from Heartcore was a single endpoint on the Delivery API. This endpoint returned paginated redirect information for _all_ content items in the tree. If you needed to find the canonical URL for a given path, then this endpoint was your only option. You would need to find your path in the returned items - especially challenging if it was not on the first page of results
```

File: umbraco-heartcore/release-notes/2024-09-releasenotes.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+
+In order to save you the trouble, a brand new endpoint has been added to the Delivery API. It allows you to fetch the canonical URL (and any other redirects) for a given path. For more information, check out the [updated page here](../api-documentation/content-delivery/redirect.md#get-content-by-redirect-url).
```
Comment (sofietoft) [inline umbraco-heartcore/release-notes/2024-09-releasenotes.md:7]:
```suggestion
In order to save you the trouble, a brand new endpoint has been added to the Delivery API. It allows you to fetch the canonical URL (and any other redirects) for a given path. For more information, check out the [updated Redirect article](../api-documentation/content-delivery/redirect.md#get-content-by-redirect-url).
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks for the PR @jmf-umbraco ! 💪 

I've made some suggestions, mainly to keep the content consistent with the rest of our docs 🙌 

Also, could I get you to add the following line to the `.gitbook.yaml` file in the umbraco-heartcore directory:

`api-documentation/redirect: api-documentation/content-delivery/redirect.md`

^ It's a redirect for the article that was moved 💪 

Finally, let me know when this should be merged - thanks!

**Comment by jmf-umbraco** [comment]
@sofietoft - Those suggestions all make sense to me, I'll commit them. And absolutely will add the redirect. I was going to ask about how we typically handle those, but I figured it would come up during review, so thanks for the info 😆

**Comment by jmf-umbraco** [comment]
Changes made. The new redirect endpoint goes out tomorrow, so any time from then onwards is fine.

**Comment by sofietoft** [comment]
Thanks for considering the suggestions @jmf-umbraco ! 😄 
I just added a missing "redirects" heading to the yaml file - I thought it was already there 😅 

But all looks good 👏 
I'll get it merged once all the checks clears!

**Review by sofietoft — APPROVED** [review]

**Pull request merged**